tests/framework/db/ActiveQueryTest.php

```diff
@@ -129,6 +129,24 @@ public function testInnerJoinWith()
         ], $result->joinWith);
     }
 
+    /**
+     * @todo: tests for the regex inside getQueryTableName
+     */
+    public function testGetQueryTableName_from_not_set()
```

Comment:
for the simple reason that multiple tests for the same method can be easily distinguished